### ISSUE TITLE
Consider current anchor

### ISSUE THREAD

**Issue by OLibutzki** [maciejwalkowiak/spring-docs-read-latest-extension#1]
Is it possible to consider the current anchor?

Like https://docs.spring.io/spring-boot/docs/2.2.0.RELEASE/reference/htmlsingle/#cloud-deployment-openshift --> https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#cloud-deployment-openshift

**Comment by maciejwalkowiak** [comment]
Current anchor should work already - your specific example works too, at least for me.. 🤔

**Comment by OLibutzki** [comment]
Yes, it works if visit the doc site with the anchor in the first place.

If you navgate and want to switch to the current docs then the link to the current docs does not consider the latest anchor.

**Comment by maciejwalkowiak** [comment]
Had a look into this, and ideally its added to the `tocbot` - the script Spring docs uses for table of contents. Otherwise URL change either gets out of sync with the table of contents, or I would have to copy quite a bit of tocbot code. Created https://github.com/tscanlin/tocbot/issues/229 - lets see how it goes.
